packages/transport/test/modules/devices/SetCommunicationLanguage.test.ts

```diff
@@ -0,0 +1,34 @@
+import { IDatabaseConnection } from "@js-soft/docdb-access-abstractions";
+import { AccountController } from "../../../src";
+import { AppDeviceTest } from "../../testHelpers/AppDeviceTest";
+import { DeviceTestParameters } from "../../testHelpers/DeviceTestParameters";
+import { TestUtil } from "../../testHelpers/TestUtil";
+
+describe("set the communication language", function () {
+    let connection: IDatabaseConnection;
+
+    let deviceTest: AppDeviceTest;
+
+    let deviceAccount: AccountController;
+
+    beforeAll(async function () {
+        connection = await TestUtil.createDatabaseConnection();
+
+        const parameters = new DeviceTestParameters({ ...TestUtil.createConfig(), datawalletEnabled: true }, connection, TestUtil.loggerFactory);
+        deviceTest = new AppDeviceTest(parameters);
+
+        await deviceTest.init();
+
+        deviceAccount = await deviceTest.createAccount();
+    });
+
+    afterAll(async function () {
+        await deviceTest.close();
+
+        await connection.close();
+    });
+
+    test("should set the communication language", async function () {
+        await expect(deviceAccount.activeDevice.setCommunicationLanguage("fr")).resolves.not.toThrow();
+    });
```

Comment:
missing a negative test here

Reply:
added the test, but it's failing @tnotheis

Reply:
You passed a 3-digit language code. That's not allowed. Only 2-digit language codes are accepted.

Reply:
Ah, sorry, I misread the code (or rather mixed up different things).

Reply:
So... I just tested it locally, and I get a 400 as expected:

![image](https://github.com/user-attachments/assets/ebefd094-e7ac-4819-b6e6-ee0f35af04a0)